packages/babel-plugin-transform-es2015-template-literals/src/index.js

```diff
@@ -18,7 +18,9 @@ export default function ({ types: t }) {
         let raw = [];
 
         for (const elem of (quasi.quasis: Array)) {
-          strings.push(t.stringLiteral(elem.value.cooked));
+          if (elem.value.cooked !== null) {
```

Comment:
Cool. Is every cooked value excluded if one is, or is it individual value either may or may not use invalid escapes independent of the others?